### PR TITLE
Bump to 0.5.1 zkverify version

### PR DESCRIPTION
**Pull request by JackPiri**
zkVerify GitHub tag 0.5.1-0.5.0 (up to 0.5.0-Infinity) zkVerify Docker tag 0.5.1

## Description
Bumping to 0.5.1

## Jira Ticket
[ZKV-522](https://horizenlabs.atlassian.net/browse/ZKV-552)

## Changes
 - Bumping versions

## Breaking Changes
None

## Checks
- [ ] Project Builds
- [ ] Project passes tests and checks
- [ ] Updated documentation accordingly
- [ ] Breaking changes have been correctly tagged and notified

## Additional information
None

### CODE REVIEW

File: CHANGELOG.md
Comment (ysibirski) [inline CHANGELOG.md:5]: We can not merge this one until the release/tag on zkVerify is ready :(I just looked and it is not yet there)

**Review by lander86 — APPROVED** [review]
LGTM ; to be merged once tag will be ready on DH

**Review by ysibirski — COMMENTED** [review]
Also you had a comment in your prev PR 
> Migration "NH_" -> "ZKV_" (TODO: remove in next release)

Do we want to remove this in this PR then ?

File: CHANGELOG.md
Comment (JackPiri) [reply CHANGELOG.md:5]: I agree, there's the last run of the CI running; I'll wait for its completion and then create tag there and then merge here.

**Review by ysibirski — APPROVED** [review]
LGTM, see some comments and question

**Comment by JackPiri** [comment]
> Also you had a comment in your prev PR
> 
> > Migration "NH_" -> "ZKV_" (TODO: remove in next release)
> 
> Do we want to remove this in this PR then ?

No let's keep it, this `0.5.1` is for fixing a sporadic issue on `0.5.0`, but the need of migrating `nh_` data directory to `zkv_` data directory is still present. This will be dropped in next release (I'd say `0.6.0` or similar)
Btw, thanks for the double check.

**Pull request merged**
